iOS Inspector_files/inspector.js

```diff
@@ -16,11 +16,20 @@
  *
  */
 
-
+const getParameterByName = (name, url) => {
+    if (!url) url = window.location.href;
+    name = name.replace(/[\[\]]/g, "\\$&");
+    var regex = new RegExp("[?&]" + name + "(=([^&#]*)|&|#|$)"),
+        results = regex.exec(url);
+    if (!results) return null;
+    if (!results[2]) return '';
+    return decodeURIComponent(results[2].replace(/\+/g, " "));
+}
 
 Inspector.logLevel = 4; // 0=none, 1=error, 2=error +warning, 3=
 // error,warning,info 4 = all
-const APPIUM_ROOT = "http://localhost:4723/wd/hub";
+const PORT = getParameterByName("port") || '4723';
+const APPIUM_ROOT = `http://localhost:${PORT}/wd/hub`;
```

Comment:
I'd rather keep it compatible to older browsers. Please use string concatenation instead